### DISCUSSION TITLE
Cannot open svg image on mobile app

### DISCUSSION THREAD

**Discussion by hongbo-miao** [community/community#31189] [Mobile] [answered]
When I click the architecture image (svg format) at
https://github.com/Hongbo-Miao/hongbomiao.com#architecture
on my GitHub mobile app.

It always shows "Something went wrong".

- GitHub mobile version: 1.77.0 (76215101)
- iOS: 15.6.1

![image](https://user-images.githubusercontent.com/3375461/187124084-bd940eaa-2695-4815-bfaa-38bb6a3e886a.png)

**Comment by eliperkins** [comment, marked as answer]
Hey @Hongbo-Miao! Thanks for this feedback.

Our engineering team took a look at this and it's definitely a bug on our side, due to the SVG being rather large (~7.5mb). Regardless, as a user, you should still be able to access this SVG if you've navigated directly to it.

We'll look to get this bug fixed within GitHub Mobile, but for now, a workaround can be to open the repository on GitHub.com to access the SVG directly. Thanks!

**Reply by hongbo-miao** [reply]
Just add a following up, as of today, it can show large SVG now on mobile app. Thanks!

**Comment by ALAWIII** [comment]
Scrolling over svg rendered images is laggy,
Android 14,
Github mobile version: 1.251.0 

Project repo where svg is located : https://github.com/ALAWIII/family-cloud

https://github.com/user-attachments/assets/ae0b8b75-0509-4f7d-b967-2d45fded502c
